src/redux/reducers/genes/downsampledGenesLoaded.js

```diff
@@ -0,0 +1,65 @@
+import _ from 'lodash';
+
+import { initialViewState } from 'redux/reducers/genes/getInitialState';
+import { upperCaseArray } from 'utils/genes';
+
+const downsampledGenesLoaded = (state, action) => {
```

Comment:
Cant we store the genes in the same place, why do we need a separate path for the downsampled ones. Can't we just have a flag downsampled: true, to judge if they are or they not and just use the same path everywhere?

Reply:
Both downsampled and full expression have sepratre `loading`, `error`, etc properties. How would you join them?

Reply:
I don't understand. loading and error can correspond to the current genes that we have. The other option is to have a separate entry in redux for downsampled or full genes errors/loading.

Reply:
> The other option is to have a separate entry in redux for downsampled or full genes errors/loading.

This is what I implemented, we have expression.full and expression.downsampled